### PR TITLE
out_secondary_file: add missing option explanation

### PR DESCRIPTION
**Pull request by daipom**
This explanation is missing since the beginning.

* https://github.com/fluent/fluentd/blob/master/CHANGELOG.md#release-v0145---20160906
* https://github.com/fluent/fluentd/pull/1154

### CODE REVIEW

**Review by ashie — APPROVED** [review]
LGTM

**Comment by daipom** [comment]
Thanks for merging!

File: output/secondary_file.md [hunk 1]
```diff
@@ -99,6 +99,16 @@ This makes parallel file processing easy. But if you want to disable this behavi
 dump.bin
 ```
 
+### `compress`
+
+| type | default | version |
+| :--- | :--- | :--- |
+| enum | text | 1.0.0 |
```
Comment (ashie) [inline output/secondary_file.md:106]: Actual version which adds this feature seems 0.14.5 but it's meaning less to specify exact version.
* https://github.com/fluent/fluentd/blob/master/CHANGELOG.md#release-v0145---20160906
* https://github.com/fluent/fluentd/pull/1154

Comment (daipom) [reply output/secondary_file.md:106]: Yes, the other options are the same.
Or maybe every option here could be changed to v0.14.5.

**Pull request merged**